spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'order'
+
+describe Order do
+  let(:time_klass){ double(:time_klass, new: 0) }
+  subject(:order) { Order.new(time_klass) }
+
+  let(:quantity){ 3 }
+  let(:price) { 1 }
+  let(:total) { quantity * price }
+  let(:name) { 'Food' }
+  let(:dish){ double(:dish, name: name, price: price)}
+  let(:receipt){ "Your order has been placed and will be delivered at 3600: " \
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.